### PR TITLE
Add simple functionality for new drawing

### LINKED ISSUE #13: New drawing should create a new drawing
## Problem
Bring functionality to the 'New drawing' menuItem.

### PR DESCRIPTION
**Pull request by tim-kilian**
Closes #13 

## Changes
- Create a new instace of moder and simulator on new drawing

## Open issues 
- Find a better solution for the setTimeout and the exchange

### CODE REVIEW

File: src/components/header/menu/dropdowns/MenuDropdownFile.vue [hunk 1]
```diff
@@ -69,6 +75,19 @@ export default {
...
+        newDrawing () {
+            this.$instances.modeler = new Modeler();
+            this.$instances.modeler.addFormalism(PluginBase);
+            this.$instances.modeler.addFormalism(PluginPT);
+            this.$instances.simulator = new Simulator();
+            this.$instances.simulator.addFormalism(PluginBase);
+            this.$instances.simulator.addFormalism(PluginPT);
+
+            this.$store.commit('setActiveContext');
+            setTimeout(() => {
```
Comment (0x4d) [inline src/components/header/menu/dropdowns/MenuDropdownFile.vue:87]: We should eventually find a way that avoids the need to do this, but fine for now :smile:

**Review by 0x4d — APPROVED** [review]
LGTM

File: src/components/header/menu/dropdowns/MenuDropdownFile.vue (hunk 1 above)
Comment (tim-kilian) [reply src/components/header/menu/dropdowns/MenuDropdownFile.vue:87]: Sry for this, but I couldn't figure it out. :( Where could we put it, so we dont forget about it? An own issue, or an existing one?

Comment (tim-kilian) [reply src/components/header/menu/dropdowns/MenuDropdownFile.vue:87]: Maybe I leave #13 open.

Comment (0x4d) [reply src/components/header/menu/dropdowns/MenuDropdownFile.vue:87]: Can you create a new issue for this? Maybe describe why you couldn't figure it out / what else you have already tried?

Comment (tim-kilian) [reply src/components/header/menu/dropdowns/MenuDropdownFile.vue:87]: I've reopend the issue already. 🗡

Comment (tim-kilian) [reply src/components/header/menu/dropdowns/MenuDropdownFile.vue:87]: But, you're right and move it to this new issue #27.

**Pull request merged**